### ISSUE TITLE
Request Network Activity feature in this tool

### ISSUE THREAD

**Issue by tlsalex** [Nuke928/NTop#4]
Today , one of user complain that why her outlook's connection so slowly.  she didn't see any new email in her mailbox . but someone already sent her many new emails in this morning.

Then I came up with an idea , will it possible to add the network usage (Receive (B/sec) Total (B/sec)) for each process in this tool.

Under the GUI of  Resource Monitor ,in the tab of Network , we can see an area called Network Activity , it list image , PID, Address, Send(B/sec), Receive(B/sec), Total(B/sec).

I hope you add  network usage {Send(B/sec), Receive(B/sec), Total(B/sec).} into your tool , maybe looks like too many feature inside this tool .

I prefer you can make these features into an individual tool , name it with Ntna or just Na (Network Activity).

![na](https://user-images.githubusercontent.com/36788337/56959691-ded50880-6b80-11e9-9812-bba00ca3ae48.png)

Thank you!

**Comment by tlsalex** [comment]
BTW , under Task Manger , we also can see the Network column,
Maybe it's more convenient  just add the network usage (total B/sec) into this tool.
As the Network Activity is too detail , better make network activity into individual tool.

**Comment by Nuke928** [comment]
I'm not interested in making an entire new tool for this. Would just one total usage column like in task manager suffice?

**Comment by tlsalex** [comment]
Yes , please try to add one more column , which is  the column of total network usage
Thank you

**Comment by tlsalex** [comment]
Hello Sir ,

Any update ?

BR

**Comment by Nuke928** [comment]
Sorry, I've been busy. I'll work on it this weekend.
